apps/fluent-tester/package.json

```diff
@@ -20,9 +20,10 @@
     "bundle": "fluentui-scripts bundle"
   },
   "dependencies": {
-    "@fluentui/react-native": ">=0.15.89 <1.0.0",
+    "@fluentui/react-native": ">=0.15.90 <1.0.0",
     "@fluentui-react-native/stack": ">=0.2.8 <1.0.0",
     "@fluentui-react-native/interactive-hooks": ">=0.5.1 <1.0.0",
+    "@fluentui-react-native/text": ">=0.6.8 <1.0.0",
```

Comment:
Lots of extra changes showing up in this review, is the merge baseline messed up in some way?  Also I would expect text to be included in @fluentui/react-native so it is unnecessary to add it as a dependency.

Reply:
I had a bunch of extra changes from dependbot for some reason. I've rebased to where master should be now. That should remove a bunch of the files changed.